ksql-engine/src/test/java/io/confluent/ksql/integration/StreamsSelectAndProjectAvroIntTest.java

```diff
@@ -0,0 +1,153 @@
+package io.confluent.ksql.integration;
+
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.test.IntegrationTest;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import io.confluent.ksql.GenericRow;
+import io.confluent.ksql.KsqlContext;
+import io.confluent.ksql.serde.DataSource;
+import io.confluent.ksql.util.OrderDataProvider;
+import io.confluent.ksql.util.SchemaRegistryClientFactory;
+
+import static org.hamcrest.CoreMatchers.equalTo;
```

Comment:
Why can't we just use `assertEquals` from junit. It _should_ cover the types being used in this test. I don't think we need this new dependency.

Reply:
We have been using `assertThat` in other tests, so just for consistency.